### DISCUSSION TITLE
在arm64架构有安装方法吗

### DISCUSSION THREAD

**Discussion by ycilry** [apache/apisix#5497] [Q&A]
arm64安装apisix折腾我好久了，各种问题，最后只能通过docker方式安装，很期待能提供arm64架构安装指导，非常感谢

**Comment by juzhiyuan** [comment]
I remember Apache APISIX supports ARM since 0.8, may I know what issues happened?

**Reply by tzssangglass** [reply]
here is: https://github.com/tzssangglass/apisix-build-tools/blob/mac_arm64/build-apisix-openresty-darwin-arm64.sh my script for building APISIX-OpenResty on the Apple M1, I hope it inspires you.
